substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -208,6 +208,23 @@ where
 	ChainApi: graph::ChainApi<Block = Block> + 'static,
 	<Block as BlockT>::Hash: Unpin,
 {
+	// Injects a view for genesis block to self.
```

Comment:
nit: can we update this? It is not necessarily a genesis block per se we could inject here, IIUC.